app/vue/src/client/preview/story_store.js

```diff
@@ -0,0 +1,89 @@
+/* eslint no-underscore-dangle: 0 */
+
+let count = 0;
+
+function getId() {
+  count += 1;
+  return count;
+}
+
+export default class StoryStore {
```

Comment:
I agree that we should do this but, given we have the same problem with the RN layer, it's a big refactor that can come later, I propose in v4: https://github.com/storybooks/storybook/issues/1354